recipes/python/retail/retail/utils.py

```diff
@@ -0,0 +1,31 @@
+#####################################################################
+# ADOBE CONFIDENTIAL
+# ___________________
+#
+#  Copyright 2017 Adobe
+#  All Rights Reserved.
+#
+# NOTICE:  All information contained herein is, and remains
+# the property of Adobe and its suppliers, if any. The intellectual
+# and technical concepts contained herein are proprietary to Adobe
+# and its suppliers and are protected by all applicable intellectual
+# property laws, including trade secret and copyright laws.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Adobe.
+#####################################################################
+
+
+class Utils:
+
+
+    def renameColumns(self, configProperties={}, dataframe=None):
```

Comment:
I dont think `configProperties` is used here.

Reply:
Suggestion for a method name: `remove_tenantId_from_column_labels`

Reply:
I'm in favor too of renaming functions to be more indicative of what they're doing.  I would suggest however to retain style - don't use underscore if camelCase is the standard.

Reply:
@rjrussell77 This is outdated commit with Utils class. I have removed that class later on, with new and better approach.